.github/scripts/addUnblockedComment.js

```diff
@@ -0,0 +1,56 @@
+// When a blocking issue is closed:
+// 1. Leaves a comment on all the issues listed as blocked in body,
+
+module.exports = async ({ github, context }) => {
+  const { ISSUE_VERIFIERS } = process.env;
+  const issueBody = context.payload.issue.body;
+  const blockedIssuesRegex = /Blocked issues:\s*(#\d+(?:,\s*#\d+)*)/i;
+
+  if (!issueBody) {
+    console.log("No issue body was found");
+    return;
+  }
+
+  // Get the list of issues blocked by this issue if there are any
+  const blockedIssuesLine = issueBody.match(blockedIssuesRegex);
+
+  // Add a "@" character to notify the user
+  const verifiers = ISSUE_VERIFIERS.split(",").map((v) => " @" + v.trim());
```

Comment:
```suggestion
  const verifiers = ISSUE_VERIFIERS.split(",").map((v) => `@${v.trim()}`);
```
Nitpick: can you use a string literal here instead of concatenating